src/screens/Home/Home.js

```diff
@@ -247,6 +260,49 @@ class HomeScreen extends React.Component<Props, State> {
     );
   };
 
+  handleReferralBannerPress = () => {
+    const { navigation, user } = this.props;
+    const { isEmailVerified, isPhoneVerified } = user;
+    if (isEmailVerified || isPhoneVerified) {
+      navigation.navigate(REFER_FLOW);
+    } else {
+      toastReferral(navigation);
+    }
+  };
+
+  renderReferral(colors) {
+    const { referralsFeatureEnabled } = this.props;
+    const { isReferralBannerVisible } = this.state;
+
+    if (!referralsFeatureEnabled) {
+      return null;
+    }
+
+    return (
+      <Banner
+        isVisible={isReferralBannerVisible}
+        onPress={this.handleReferralBannerPress}
+        bannerText="Refer friends and earn rewards, free PLR and more."
+        imageProps={{
+          style: {
+            width: 96,
+            height: 60,
+            marginLeft: 4,
+          },
+          source: referralImage,
+        }}
+        wrapperStyle={{ borderBottomWidth: 1, borderBottomColor: colors.border }}
+        onClose={() => this.setState({ isReferralBannerVisible: false })}
+      />
+    );
+  }
```

Comment:
Just to follow the pattern we use throughout existing components.
```suggestion
  renderReferral = (colors) => {
    const { referralsFeatureEnabled } = this.props;
    const { isReferralBannerVisible } = this.state;

    if (!referralsFeatureEnabled) {
      return null;
    }

    return (
      <Banner
        isVisible={isReferralBannerVisible}
        onPress={this.handleReferralBannerPress}
        bannerText="Refer friends and earn rewards, free PLR and more."
        imageProps={{
          style: {
            width: 96,
            height: 60,
            marginLeft: 4,
          },
          source: referralImage,
        }}
        wrapperStyle={{ borderBottomWidth: 1, borderBottomColor: colors.border }}
        onClose={() => this.setState({ isReferralBannerVisible: false })}
      />
    );
  };
```